lib/src/DOtherSide.cpp

```diff
@@ -259,15 +259,15 @@ void dos_qqmlapplicationengine_load_data(::DosQQmlApplicationEngine *vptr, const
     engine->loadData(data);
 }
 
-void dos_qapplication_load_translation(::DosQQmlApplicationEngine *vptr, const char* translationPackage)
+void dos_qapplication_load_translation(::DosQQmlApplicationEngine *vptr, const char* translationPackage, bool shouldRetranslate)
 {
     if (!m_translator->isEmpty()) {
         QCoreApplication::removeTranslator(m_translator);
     }
     if (m_translator->load(translationPackage)) {
         bool success = QCoreApplication::installTranslator(m_translator);
         auto engine = static_cast<QQmlApplicationEngine *>(vptr);
-        engine->retranslate();
+        if (shouldRetranslate) engine->retranslate();
```

Comment:
Do we need to load the translation if we don't retranslate? Can we return early?